src/packages/media/media-types/index.ts

```diff
@@ -1,10 +1,12 @@
 import './components/index.js';
 
+export type { UmbAllowedMediaTypeModel } from './repository/index.js';
 export {
```

Comment:
Consider if we should just export everything from the repository/index.js file instead.